RoboFile.php

```diff
@@ -559,6 +573,17 @@ protected function getPluginVersion($file) {
     return !empty($m[1]) ? trim($m[1]) : false;
   }
 
+  protected function getChangelogController() {
+    require_once './tasks/release/ChangelogController.php';
+    $this->loadEnv();
+    return \MailPoet\Release\ChangelogController::getInstance(
```

Comment:
Namespace `MailPoet` is PSR-4-loaded from `lib`. We should use a different namespace.